e2e/cli_test.go

```diff
@@ -1083,7 +1072,7 @@ Cannot move multiple files to: bar/bar.rego
 	}
 
 	for file, expectedContent := range initialState {
-		bs := testutil.Must(os.ReadFile(filepath.Join(td, file)))(t)
+		bs := testutil.MustReadFile(t, filepath.Join(td, file))
```

Comment:
We could even consider some custom linters for this kind of thing... 🤔

Reply:
Just remove `testutil.Must` and the compiler is your linter 🙃 On a more productive note, https://github.com/quasilyte/go-ruleguard might be your friend.

Reply:
Ooh, that looks like a nice project!

Reply:
Pretty sure the compiler enforces types for imported testutil packages as well ;)